web/packages/teleport/src/JoinTokens/JoinTokens.tsx

```diff
@@ -76,8 +81,7 @@ export const JoinTokens = () => {
     { join_token: '' } // we are only editing for now, so template can be empty
   );
 
-  async function handleSave(content: string): Promise<void> {
-    const token = await ctx.joinTokenService.upsertJoinToken({ content });
+  function updateTokenList(token: JoinToken): JoinToken[] {
```

Comment:
does line 97 still apply? (since we prevent user from changing the name on edit now)